### PR TITLE
First version of zabbix alert

### PR DESCRIPTION
**Pull request by alvarolmedo**
This alert send metric by hit to a pair zbx_host/zbx_item configured in the alert.
`alert:
  - zabbix.ZabbixAlerter:
      output_file_path: "/tmp/alerts.log"
      zbx_sender_host: zabbix-server
      zbx_sender_port: 10051
      zbx_host: host
      zbx_key: item
`

### CODE REVIEW

File: elastalert/zabbix.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
+from alerts import Alerter  # , BasicMatchString
+import logging
+# from pprint import pprint
+from pyzabbix.api import ZabbixAPI
+from pyzabbix import ZabbixSender, ZabbixMetric
+# from pyzabbix.api import ZabbixAPIException
+# from requests.packages.urllib3.exceptions import InsecureRequestWarning
```
Comment (Qmando) [inline elastalert/zabbix.py]: Can you remove all the commented out imports?

File: elastalert/zabbix.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        self.aggregated_metrics = []
+        self.logger = logging.getLogger(self.__class__.__name__)
+        super(ZabbixClient, self).__init__(url=self.url, use_authenticate=self.use_authenticate, user=user, password=password)
+
+    def send_metric(self, hostname, key, data):
+        zm = ZabbixMetric(hostname, key, data)
+        if self.send_aggregated_metrics:
+
+            self.aggregated_metrics.append(zm)
+            if len(self.aggregated_metrics) > self.metrics_chunk_size:
```
Comment (Qmando) [inline elastalert/zabbix.py]: What happens if this *isn't* true?

File: elastalert/zabbix.py (hunk 1 above)
Comment (alvarolmedo) [reply elastalert/zabbix.py]: Removed

File: elastalert/zabbix.py (hunk 2 above)
Comment (alvarolmedo) [reply elastalert/zabbix.py]: The metric is appended (line 28, line 25 after remove imports commented), it isn't lost. When the number of metrics appended is greather than the metrics_chunk_size, the metrics are sent by ZabbixSender (line 32, line 29 after remove imports commented)

Comment (alvarolmedo) [reply elastalert/zabbix.py]: @Qmando what do you think about the PR? maybe I can setup a var to manage the metrics_chunk_size in order to control the shipments to zabbix with it.

**Comment by tetebueno** [comment]
Any news on when this will merged?
Cheers.

File: elastalert/zabbix.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    # Alert is called
+    def alert(self, matches):
+
+        # Matches is a list of match dictionaries.
+        # It contains more than one match when the alert has
+        # the aggregation option set
+
+        zm = []
+
+#        zm =  [ZabbixMetric(self.zbx_host, self.zbx_key, matches_len)]
```
Comment (Qmando) [inline elastalert/zabbix.py]: What's with all the commented out code?

**Comment by Qmando** [comment]
Happy to merge once commented code is removed and documentation added

**Comment by alvarolmedo** [comment]
Commented code removed, doc added and all checks have passed. :) Thanks @Qmando

**Comment by alvarolmedo** [comment]
@Qmando has the next release of elastalert date to be published?

**Comment by alvarolmedo** [comment]
@Qmando is the doc provided enough?

**Pull request merged**